packages/query/package.json

```diff
@@ -22,23 +22,32 @@
   },
   "bugs": "https://github.com/leather-wallet/mono/issues",
   "dependencies": {
+    "@fungible-systems/zone-file": "2.0.0",
```

Comment:
If this is being installed just for `parseZoneFile`, it might be we could just add that functionality directly and get rid of this? We appear to only use it in `bns.utils.ts`.